docker/Dockerfile_ubuntu_16.04

```diff
@@ -0,0 +1,27 @@
+FROM nvidia/cudagl:9.0-runtime-ubuntu16.04
```

Comment:
I vote we get rid of the 16.04 image, I don't see a point

Reply:
I could see this being important if there were compatibility concerns with existing customers, but I agree with Jayden here.

Reply:
Done